packages/libp2p/src/connection-manager/connection-pruner.ts

```diff
@@ -1,6 +1,8 @@
 import { PeerMap } from '@libp2p/peer-collections'
 import { safelyCloseConnectionIfUnused } from '@libp2p/utils/close'
+import { convertToIpNet } from '@multiformats/multiaddr/convert'
 import { MAX_CONNECTIONS } from './constants.js'
```

Comment:
Use `multiaddrToIpNet` instead:

```suggestion
import { MAX_CONNECTIONS } from './constants.js'
import { multiaddrToIpNet } from './utils.js'
```